docs/01-overview/main-areas/serverless/svls-01-function-deployment-options.md

```diff
@@ -0,0 +1,69 @@
+Focus on the function code and develop it from you favourite IDE. Configure your functions directly in the [`config.yaml` manifest file](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-06-function-configuration-file/)
+>>NOTE: Use `kyma init function --vscode` to generate json schema which can be used in VScode for autocompletion.
+>>
+
+Kyma CLI helps you run your code locally with a single `kyma run function` command. It runs the function using your local docker deamon with the same runtime docker context as if it was run in the kyma runtime. 
+>>NOTE: Use `kyma run function` with `--hot-deploy` and spare yourself unnecessary restarts of the functions whenever you test a changed function logic. Also, use [`--debug` option](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-05-debug-function) to allow connecting with your favourite debugger.
+>>
+
+![kyma-cli-functions](../assets/kyma-cli-functions.png)
+
+Having written and tested your function locally, simply deploy it to the kyma runtime with `kyma apply function` command used in the folder of your function's workspace. It will read the files, translate it to kubernetes manifests and deploy the function.
+
+
+# Deploy via CI/CD
+
+Kyma UI helps you getting started. Kyma CLI helps you iterate and develop functions. 
+But at the end of the day you probably want an automated deployment of your functions. Or rather, of your whole application, where functions are just part of it.
+It all comes down to being able to deploy k8s applications on different kyma runtimes in a GitOps fashion and, for the sake of simplicity, the deployment approach for functions should not differ from deployment of other kubernetes workloads, config-maps or secrets.
+
+So in the end what you need is those yaml manifests for everything - including functions.
+
+Fortunatelly, Kyma CLI helps you generate the yaml mainfests matching your `config.yaml` file you crafted in earlier phase.
+Use `--dry-run` option of the `kyma apply function` command to generate kubernetes manifests that will include the function CR itself but also all the related CRs (i.e ApiRules, Subscriptions, etc).
+
+```bash
+kyma apply function --dry-run --ci -o yaml > my-function.yaml
+```  
+
+The generated manifest should be part of all the manifests that define your application and pushed to the git repository.
+Deploy everything in a consistent way either via CI/CD or via GitOps operators ( i.e [fluxcd](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-06-sync-function-with-gitops/) ) installed in your kyma runtime.
+
+>>NOTE: Source Function Code  directly from Git Repository
+
+Kyma Functions come in two types : `git` and `inline`.
+[Git type](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-serverless/svls-02-create-git-function/) allows you to configure a git repository as a source of your function code instead of creating it "inline".
```

Comment:
```suggestion
[Git type](../../03-tutorials/00-serverless/svls-02-create-git-function.md) allows you to configure a git repository as a source of your Function code instead of creating it `inline`.
```